builtin/providers/aws/cloudfront_distribution_configuration_structure_test.go

```diff
@@ -44,6 +49,35 @@ func trustedSignersConf() []interface{} {
 	return []interface{}{"1234567890EX", "1234567891EX"}
 }
 
+func lambdaSetHash(v interface{}) int {
```

Comment:
@MrGossett So.... funny enough, I had to reinstate that lambda set hash you were asked to remove 😆  

From Martin's comment:

> We only really continue to support explicit Set for compatibility with schemas that predate the automatic hashing.

This is _true_, but unfortunately `lambda_function_association` is being added as a sub param to two params that do themselves have custom hash function for the IDs. 

So, I had to add `lambda_function_association` to the two `CacheBehaviorHash` (?) method(s), but to do so, I'd need to hash them... so I needed that method 😐 

So anyway it's re-added